core/frontend/src/components/vehiclesetup/configuration/compass/AutoCoordinateDetector.vue

```diff
@@ -139,6 +171,15 @@ export default {
           },
         },
       )
+      await sleep(0.5)
```

Comment:
Can we have a scope defined function to avoid the duplicated code ?

Reply:
done